docs/types/txikijs.d.ts

```diff
@@ -374,14 +374,7 @@ declare global {
         *
         * @param path Path to convert.
         */
-        function realpath(path: string): Promise<string>;
-        
-        /**
-        * Removes the given file.
-        *
-        * @param path Path to be removed.
-        */
-        function unlink(path: string): Promise<void>;
```

Comment:
There is a difference between `remove` and `unlink` which most people would not be able to state without checking the docs :D. So it does make sense to remove one & simplify the interface since in the vast majority of cases one is redundant, but I just wanted to make sure this was an "informed choice".

Reply:
Yes this is intentional. It's an extension of remove, which supports recursive deletion, off by default.